src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonPage } from './PersonPage';
+
+export const PeopleTable = ({ people }: { people: Person[] }) => {
+  const { personId = '' } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
The names of the columns can be written in an array and re-rendered through the map.